src/js/components/BannerDeadline.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import moment from 'moment'
+import * as dates from '../constants/dates.js'
+
+export const withinAWeekOfDeadline = (today, filingPeriod) => {
+  if(!moment(today).isValid()) {
+    return false
+  }
+
+  if(moment(today).isBetween(`${filingPeriod}-${dates.ONE_WEEK_TO_FILE}`, `${filingPeriod}-${dates.FILING_DEADLINE}`, 'day', '[]')) {
+    return true
+  }
+  return false
+}
+
+const BannerDeadline = (props) => {
+  const today = moment().format('YYYY-MM-DD')
+
+  // render if within the filing period (1/1/<filingPeriod> to 3/1/<filingPeriod>)
+  if(moment(today).isBetween(`${props.filingPeriod}-${dates.FILING_START}`, `${props.filingPeriod}-${dates.FILING_DEADLINE}`, 'day', '[]')) {
```

Comment:
Won't this always be false because `today` will be the year *after* the filing period? The check should be on filingPeriod + 1